resources/qml/Preferences/Materials/MaterialsView.qml

```diff
@@ -67,400 +68,419 @@ TabView
         }
     }
 
-    Tab
+    UM.TabRow
     {
-        title: catalog.i18nc("@title", "Information")
-
-        anchors.margins: UM.Theme.getSize("default_margin").width
-
-        ScrollView
+        id: pageSelectorTabRow
+        UM.TabRowButton
+        {
+            text: catalog.i18nc("@title", "Information")
+            property string activeView: "information" //To determine which page gets displayed.
+        }
+        UM.TabRowButton
         {
-            id: scrollView
-            anchors.fill: parent
-            horizontalScrollBarPolicy: Qt.ScrollBarAlwaysOff
-            flickableItem.flickableDirection: Flickable.VerticalFlick
-            frameVisible: true
+            text: catalog.i18nc("@label", "Print settings")
+            property string activeView: "settings"
+        }
+    }
 
-            property real columnWidth: (viewport.width * 0.5 - UM.Theme.getSize("default_margin").width) | 0
+    ScrollView
+    {
+        id: informationPage
+        anchors
+        {
+            top: pageSelectorTabRow.bottom
+            left: parent.left
+            right: parent.right
+            bottom: parent.bottom
+        }
 
-            Flow
+        ScrollBar.vertical: UM.ScrollBar
+        {
+            parent: informationPage
+            anchors
             {
-                id: containerGrid
-
-                x: UM.Theme.getSize("default_margin").width
-                y: UM.Theme.getSize("default_lining").height
+                top: parent.top
+                right: parent.right
+                bottom: parent.bottom
+            }
+        }
+        ScrollBar.horizontal.policy: ScrollBar.AlwaysOff
+        clip: true
+        visible: pageSelectorTabRow.currentItem.activeView === "information"
```

Comment:
I did this to keep it simple. One alternative would be to use a stack view, or a loader of which we change the source. All good options, but since it's only 2 tabs here I figured this would be the easiest way to make it work. For a stackview you'd have to use an `onCurrentIndexChanged` to change the view along with the tabs. For a loader you'd have to do something similar but change the component being loaded.
The preferences pages as a whole use a StackView to change the page. The PPA uses a loader iirc.

Would you prefer one of those solutions?

Reply:
Not if this is the only place, no. Just something to keep in mind if we run into this more often.